RELEASE/scripts/autoscend/auto_buff.ash

```diff
@@ -731,12 +740,15 @@ boolean buffMaintain(effect buff, int mp_min, int casts, int turns, boolean spec
 			useSkill = $skill[Tenacity of the Snapper];
 		}																						break;
 	case $effect[There is a Spoon]:				useItem = $item[Dented Spoon];					break;
+	case $effect[They\'ve Got Fleas]:			useItem = $item[Out-of-work circus flea];		break;
```

Comment:
Add this to the beaten up handler. Search for `buffMaintain($effect[Everything Is Bananas]);` to find the spot. Probably put it outside this if statement though. It is there so we don't block gremlin tool usage, but don't need that check for the flea potion. It doesn't block, just delevels.